src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,238 @@
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string IfaceName
+        {
+            get
+            {
+                string ifaceName;
+                int ret = Interop.Stc.Info.GetIfaceName(_infoHandle, out ifaceName);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get interface name from info, Error - " + (StcError)ret);
+                    return "";
+                }
+                return ifaceName;
+            }
+        }
+
+        /// <summary>
+        /// Get the time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <param name="from">The beginning of the time interval.</param>
+        /// <param name="to">The end of the time interval.</param>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        public void GetTimeInterval(out long from, out long to)
+        {
+            int ret = Interop.Stc.Info.GetTimeInterval(_infoHandle, out from, out to);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get time interval from info, Error - " + (StcError)ret);
+                StcErrorFactory.ThrowStcException(ret);
+            }
+        }
+
+        /// <summary>
+        /// A property to get the interface type from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <value>Interface type.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public StcIfaceType Iface
+        {
+            get
+            {
+                StcIfaceType iface;
+                int ret = Interop.Stc.Info.GetIface(_infoHandle, out iface);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Interface type from info, Error - " + (StcError)ret);
+                }
+                return iface;
+            }
+        }
+
+        /// <summary>
+        /// Get the counters from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
```

Comment:
Current Platform version is 5.5 and API version is 6. Please change it to <since_tizen> 6 </since_tizen>

Reply:
done